crates/holochain/src/core/workflow/validation_receipt_workflow.rs

```diff
@@ -77,15 +80,14 @@ pub async fn validation_receipt_workflow(
         if matches!(receipt.validation_status, ValidationStatus::Rejected) {
             // Block BEFORE we integrate the outcome because this is not atomic
             // and if something goes wrong we know the integration will retry.
-            conductor
-                .block(Block::new(
-                    BlockTarget::Cell(
-                        CellId::new((*dna_hash).clone(), author.clone()),
-                        CellBlockReason::InvalidOp(receipt.dht_op_hash.clone()),
-                    ),
-                    InclusiveTimestampInterval::try_new(Timestamp::MIN, Timestamp::MAX)?,
-                ))
-                .await?;
+            apply_block(Block::new(
+                BlockTarget::Cell(
+                    CellId::new((*dna_hash).clone(), author.clone()),
+                    CellBlockReason::InvalidOp(receipt.dht_op_hash.clone()),
+                ),
+                InclusiveTimestampInterval::try_new(Timestamp::MIN, Timestamp::MAX)?,
+            ))
+            .await?;
```

Comment:
Similarly, commenting lower on

```rust
if let Err(e) = holochain_p2p::HolochainP2pDnaT::send_validation_receipt(
```

This ignores timeout errors. This is fine but quite slow and not really fair processing wise. If we're receiving ops from a node that has gone offline then we should ignore other ops from it for this workflow run and talk to responsive nodes. 

The other consequence of this is coming from the behavior of Kitsune. It will retry sending for 60s. This is sort of okay but if we have a batch of ops to send receipts for then this can stack up for quite a long time and prevent the workflow from being as responsive as it should be for other apps.

I'd like to switch this over to use a network notify rather than waiting for a response. That addresses a TODO. I think it would make sense to wait for the first notify per agent to leave the box. If it goes then fire and forget the rest. Otherwise skip all receipts for that author in this workflow run.